source/pgg_mode/PGGWorld.h

```diff
@@ -20,44 +20,49 @@ class PggWorld : public SymWorld {
    *
    * Output: None
    *
-   * Purpose: To destruct the data nodes belonging to PggWorld to conserve memory.
+   * Purpose: To destruct the data nodes belonging to PGGWorld to conserve memory.
    */
-  ~PggWorld(){
-      if (data_node_Pgg) data_node_Pgg.Delete();
+  ~PGGWorld(){
+      if (data_node_PGG) data_node_PGG.Delete();
   }
 
-  /**
-   * Input: The address of the string representing the file to be
-   * created's name
-   *
-   * Output: The address of the DataFile that has been created.
-   *
-   * Purpose: To set up the file that will be used to track
-   * information about the PGG symbiont's interaction values
-   */
+   /**
+    * Input: The address of the string representing the file to be
+    * created's name
+    *
+    * Output: The address of the DataFile that has been created.
+    *
+    * Purpose: To set up the file that will be used to track mean
+    * information about the PGG symbionts in the world.
+    * This includes: (1) their total count, (2) the counts
+    * of the free and hosted symbionts, (3) the average donation values
+    * for all symbionts, and (4) the histogram data for the distribution of
+    * symbionts at various donation values.
+    */
   emp::DataFile & SetupPGGSymIntValFile(const std::string & filename) {
     auto & file = SetupFile(filename);
     auto & node1 = GetSymCountDataNode();
     auto & node2 = GetCountFreeSymsDataNode();
     auto & node3 = GetCountHostedSymsDataNode();
     auto & node4 = GetPGGDataNode();
-    node4.SetupBins(0, 1.1, 11); //Necessary because range exclusive
+    node4.SetupBins(0.0, 1.1, 10); //Necessary because range exclusive
```

Comment:
I think this should still have 11 as the final argument so that the bins are each .1 in size